### PR TITLE
Glossary updated with SORMAS Dashboard

### PR DESCRIPTION
**Pull request by BasilKaburi**

### CODE REVIEW

**Comment by JonasCir** [comment]
@BasilKaburi Thank you very much! Please introduce line breaks to make review easier! Thanks :)

File: SORMAS DASHBOARD.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## DASHBOARD
+
+### Brief Notes
+
+Epidemiologic dashboards are digital innovations for realtime and periodic concurrent visualization of multiple streams of related data for purposes of surveillance and outbreak management. By these attributes, dashboards facilitate informed decision making using multiple combinations of data sources and analyses. In this way, dashboards improve situational awareness - especially for ongoing oubreak responses. Aside providing visuals for speedy situational assessment, epidemiologic dashboards reduce information floods that may result from multiple sources of data, and minimize workflow interruptions eg. by having to manually perform periodic analyses of incoming data.
```
Comment (JonasCir) [inline SORMAS DASHBOARD.md:5]: Please split this into several lines

File: SORMAS DASHBOARD.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+**Overview of Dashboard Directory on SORMAS**
+
+The SORMAS dashboard is the default interface/entity on logging onto SORMAS. The SORMAS dashboard has two sections viz. surveillance and contacts; each displayed at a time. These are represented by tabs on the upper right corner of the screen. There are multiple filters for selecting variables of interest eg. disease, time, etc.
```
Comment (JonasCir) [inline SORMAS DASHBOARD.md:9]: same

File: SORMAS DASHBOARD.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+***Surveillance Dashboard\***
+
+The Surveillance dashboard has two sections. The first section displays summary information on counts of each infectious disease for specified times or periods. This is accompanied by graphical display of the number of cases (counts) between any two specified times or periods. The second section displays an epidemiological curve accomapnied by a case status map. Both of these have filters for displaying information of speific diseases, times, and places.
```
Comment (JonasCir) [inline SORMAS DASHBOARD.md:13]: same

File: SORMAS DASHBOARD.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+***Contact Dashboard\***
+
+The contact dashboard displays summary information of individuals reported to have come into contact with suspected or confirmed cases of an a given infectious disease. For a selceted disease of interest, the types of contact information displayed include: counts of contacts, follow-up status chart per unit time, geographical distribution (interractive map) that is able to reveal more detailed and specific information of contacts.
```
Comment (JonasCir) [inline SORMAS DASHBOARD.md:17]: same

**Review by JonasCir — CHANGES_REQUESTED** [review]

**Comment by bernardsilenou** [comment]
All is fine as I can see on the main link. I would close it for now

**Pull request closed without merging**